static/js/scroll_bar.js

```diff
@@ -46,7 +46,6 @@ export function initialize() {
         $(".fixed-app").css("left", "-" + sbWidth + "px");
         $(".fixed-app .column-middle").css("margin-left", 7 + sbWidth + "px");
 
-        $(".column-right").css("right", sbWidth + "px");
```

Comment:
This commit seems to be accidentally included, as I think it's part of another PR?  Can you remove it from this one?

Reply:
Yeah I'll do that